sdk/storage/storage-blob/package.json

```diff
@@ -82,7 +83,7 @@
     "prebuild": "npm run clean",
     "pretest": "npm run build:test",
     "test:browser": "karma start --single-run",
-    "test:node": "cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\": \\\"commonjs\\\"}\" nyc mocha --compilers ts-node/register --require source-map-support/register --reporter mocha-multi-reporters --reporter-options configFile=mocha.reporter.config.json --full-trace --no-timeouts test/*.test.ts test/node/*.test.ts",
+    "test:node": "nyc mocha --reporter mocha-multi-reporters --reporter-options configFile=mocha.reporter.config.json -t 120000 dist-test/index.node.js",
```

Comment:
How does coverage report look like? Do covered lines point to TS source code or compiled JS codes?

Reply:
Addressed in the other thread.
https://github.com/Azure/azure-sdk-for-js/pull/2297#discussion_r281833554